apis/python/tests/test_basic_anndata_io.py

```diff
@@ -85,17 +85,18 @@ def h5ad_file_X_none(request):
     "X_kind",
     [tiledbsoma.SparseNDArray, tiledbsoma.DenseNDArray],
 )
-def test_import_anndata(adata, ingest_modes, X_kind):
-    original = adata.copy()
-    adata = adata.copy()
+# Magical conftest_adata fixture
+def test_import_anndata(conftest_adata, ingest_modes, X_kind):
```

Comment:
I've replaced `conftest_` with more descriptive prefixes in #2529, and struck the `Magical conftest_adata fixture` comments (per reasoning above).

Reply:
> Defining fixtures in conftest.py is a relatively core pytest pattern ([docs](https://docs.pytest.org/en/8.2.x/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files)), and the more unique/descriptive names in https://github.com/single-cell-data/TileDB-SOMA/pull/2529 should make it easy for people to make the connection, in the worst case

> IntelliJ

> I'd rather strike a balance where this pytest "feature" is reasonably discoverable, without scare-{naming,commenting} them at every use-site

@ryan-williams thanks for your perspective. Here is where we differ:

I'm disinterested in biasing toward individual toolchains (e.g. IntelliJ). Also since we are multi-lingual it's really crucial that we bias toward less Python expertise. Whatever may seem "standard" to highly evolved Python devs such as yourself -- whom I admire & envy -- there are lots of folks "dipping in" to Python and we need to lower the barrier to entry. I don't find these to be "scare" names -- rather, this is simple inclusivity.